lib/web_ui/dev/steps/run_tests_step.dart

```diff
@@ -72,6 +72,8 @@ class RunTestsStep implements PipelineStep {
 
     final TestsByRenderer sortedTests = sortTestsByRenderer(testFiles);
 
+    int exitCode = 0;
```

Comment:
Maybe just a `bool didFail = false;` here would be a little bette. The bitwise-anded expression has no real meaning other than "zero" or "not zero" anyway

Reply:
Done